src/esm_runscripts/filedicts.py

```diff
@@ -130,8 +147,9 @@ def __init__(self, full_config: dict, attrs_address: dict):
         if self["type"] not in ["restart", "outdata"]:
             self["name_in_work"] = self.get("name_in_work", self["name_in_computer"])
 
-        # Complete paths for all possible locations
-        self._resolve_paths()
+        # initialize the locations and complete paths for all possible locations
+        locations = dict.fromkeys(location_keys, None)
```

Comment:
Is this ever actually used?

Reply:
I don't think this is needed, and it seems redundant to me. All we need is in `self._resolve_paths`

Reply:
Forgot to put the `self.`